docs/instructor/autograde.md

```diff
@@ -179,10 +181,20 @@ AutoTest also inputs two environment variables that you can use to customize or
 - Input Environment Variables
   - **ASSIGNMENT**: string = The name of the deliverable
   - **EXEC_ID**: string = The Docker container execution ID (long random SHA)
+  - **INPUT**: string = JSON stringified input data
+    - **ContainerInput** [Schema](https://github.com/ubccpsc/classy/blob/master/packages/common/types/ContainerTypes.ts#L14-L21)
+      - **CommitTarget** [Schema](https://github.com/ubccpsc/classy/blob/master/packages/common/types/ContainerTypes.ts#L36-L59)
+      - **AutoTestConfigTransport** [Schema](https://github.com/ubccpsc/classy/blob/master/packages/common/types/PortalTypes.ts#L183-L230)
 
 ### Container Output Details
 
-While AutoTest provides input for your AutoGrade container, it is the responsibility of the instructor to provide output data for AutoTest to consume and persist. Data that is not properly managed will BE LOST FOREVER after a grading run is completed. At a bare minimum, the output data must contain a report.json file that tells AutoTest the grade results of the container run. A `stdio.txt` file will be produced automatically by AutoTest. If good log information is output to the console, AutoTest will automatically record the output data and store it for you. However, you may also choose to implement custom logging and output it to the /output/admin or /output/staff directories.
+Container output is data produced by a grading run that a course decides to retain. Output data can be accessed by the URL convention: `https://{course}.students.cs.ubc.ca/portal/resource/{$EXEC_ID}/{admin, staff, or admin}/{filename.xyz}`.
+
+ie. `https://cs999.students.cs.ubc.ca/portal/resource/08dcb3a38c6aa3ccc556385b87cc13c8d3e0baf5-lab2/student/additional_feedback.txt`.
+
+The URL is a protected route accessible by authorized users. A student may only access files from the `student` folder, while admins may access all folders, etc.
```

Comment:
One question this raises is: since the path doesn't contain a student or group ID, I wonder if its protecting one students results from another? Seems like the answer is no, and we're relying on hashes being obscure.

Reply:
Although, that's a separate issue; this documentation is definitely improved.